src/peergos/server/tests/CborObjects.java

```diff
@@ -18,6 +18,7 @@
 
     @Test
     public void dosCborObject() throws Throwable {
+        if(true)throw new RuntimeException();
```

Comment:
temporary?

Reply:
yep, will revert once I check it works on github